chore: add issue template (#111)
CMS-Enterprise/sbom-harbor-ui: .github/ISSUE_TEMPLATE/propose-an-idea.md

---
name: Propose an Idea
about:

>   Used to document discussion around proposed ideas when deciding whether to   approve

Reviewer: ```suggestion
  Propose an idea to be added to the project  determining their suitability for the project roadmap.
```

  them for the roadmap.
title: ''

> labels: ''

Reviewer: See [labels/proposed](https://github.com/CMS-Enterprise/sbom-harbor-ui/labels/proposed)

```suggestion
labels: 'proposed'
```

assignees: ''
---

**Target Audience**

> The primary audience for this request is the SaaS Governance team that manually receives SBOMs. In addition, other teams (including Harbor developers) will also be uploading and viewing SBOMs.

Reviewer: ```suggestion

<!-- Describe the target audience for this change to the project. -->
```

**What’s the Value**

> Initial method to see existing SBOMs in Harbor.

Reviewer: ```suggestion

<!-- Describe the value of implementing the proposed idea. -->
```

**Details**

> More to come.

Reviewer: ```suggestion
<!-- Provide more context and details for the proposal. -->
```

**Definition of Done**

> Definition of the outcomes, that once achieved, mean the work for the proposed idea has been completed.

Reviewer: ```suggestion

<!-- Specify the outcomes that, once achieved, mean the work for the proposed idea has been completed. -->
```
